activation/wire/wire_v2_test.go

```diff
@@ -16,13 +16,15 @@ func Benchmark_ATXv2ID(b *testing.B) {
 	f := fuzz.New()
 	b.ResetTimer()
 
+	var atx *ActivationTxV2
 	for i := 0; i < b.N; i++ {
 		b.StopTimer()
-		atx := &ActivationTxV2{}
+		atx = &ActivationTxV2{}
 		f.Fuzz(atx)
 		b.StartTimer()
 		atx.ID()
 	}
+	_ = atx
```

Comment:
Is it needed to prevent optimizations?

Reply:
I was testing it. `atx` is passed to `f.Fuzz`, which already prevents inlining so this isn't needed.